src/common/components/BidInput/BidInput.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Input from '../Input'
+
+export type TBidInput = {
+  onBidChange(value: string): void
+  bid?: string
+}
+
+const BidInput: React.FC<TBidInput> = ({ onBidChange, bid }) => {
+  const [fee, setFee] = React.useState(0)
+  const handleBid = (e: React.ChangeEvent<HTMLInputElement>) => {
+    onBidChange(e.target.value)
+    setFee(Number(e.target.value) * 5)
```

Comment:
Where is this calculation logic from? Need to confirm with Jeff and implement the logic or make the function as param.

Reply:
That is only the form style. The calculation logic would be changed later.